src/translations/translationHelper.js

```diff
@@ -0,0 +1,49 @@
+import _ from 'lodash';
+
+const isRootLocaleComment =
+  reply => (reply.depth === 1 && reply.body.length === 4 && reply.body.substring(0, 2) === '# ');
+
+export const toObject = (body) => {
+  const object = {};
+  const lines = body.split(/\n/);
+  lines.forEach((line) => {
+    const message = line.split(':');
+    if (message.length === 2) {
+      object[message[0].trim()] = message[1].trim();
+    }
+  });
+  return object;
+};
+
+export const getMessages = (replies) => {
```

Comment:
It's already been sorted by `net_rshares`. Number of vote or voter dont matter and should not. The latest comment will not overwrite the translation, unless it has more `net_rshares` than others translations.. If there is 3 comments root with `# FR` all nested comments will be merged together and only the best net_rshares translations would be chosen. I don't see how it's vulnerable to attacks.